user_stories.md

```diff
@@ -0,0 +1,21 @@
+#User Stories
+
```

Comment:
User stories all need a "so that" clause to give them meaning and a target to hit.

Reply:
i.e.

as a focus group participant, I would to see 3 different images after I choose a preference so that I don't see the same image twice in a row.